resdk/resources/collection.py

```diff
@@ -28,11 +28,14 @@ class BaseCollection(BaseResource):
     WRITABLE_FIELDS = ('data', 'description', 'settings', 'descriptor_schema',
```

Comment:
OK. Did you also remove it from the list of fields? It should not be treated as a field, because fields are handled by `.save()`, `.update()`... methods and this lazy field has other logic.